www-servers/nginx/additional-files/setup-nginx.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# Create the nginx user if it doesn't exist
+if ! id "nginx" &>/dev/null; then
+	useradd -d /system/settings/nginx/ -s /bin/false -n "nginx user" nginx
+	notify --icon /system/kernel_`uname -p` \
+		"Created user: nginx"
+fi
+
+# Create the nginx group if it doesn't exist
+if ! grep -q -E "^nginx:" /etc/group; then
+	groupadd nginx
+	notify --group nginx \
+		--icon /system/kernel_`uname -p` \
+		"Created group: nginx"
+fi
+
+mkdir -p var/www/nginx
+
+if [ ! -e var/www/nginx/index.nginx.html ]; then
+	ln -s /system/documentation/packages/nginx/html/index.html var/www/nginx/index.nginx.html
```

Comment:
On 32bit this path creates broken links as they are installed in secondary architecture:

``` 	ln -s /system/documentation/packages/nginx_x86/html/index.html var/www/nginx/index.nginx.html```

Reply:
Thanks. I'll build in a check for 32bit.

Reply:
A possible solution could be to use a variable like they do for the rdef files (in the recipe)
```
local DOCDIR="$docDir"
sed -i "s|@DOCDIR@|$DOCDIR|g" $portDir/additional-files/setup-nginx.sh
```

Reply:
I went with the sed/DOCDIR approach. I'm still not able to get a 32bit VM running but hopefully this works.

Reply:
> I went with the sed/DOCDIR approach

Did the same here, need to recheck but so far it's looking ok to me :)